system/doc/reference_manual/processes.xml

```diff
@@ -109,23 +185,508 @@ spawn(Module, Name, Args) -> pid()
+	  and resources held via NIF
+	  <seecref marker="erts:erl_nif#resource_objects">resource
+	  objects</seecref> on the heap.
+	</p>
+      </section>
+    </section>
+
+    <section>
+      <marker id="signal-delivery"/>
+      <title>Delivery of Signals</title>
+      <p>
+	The amount of time that passes between the time a signal is sent and
+	the arrival of the signal at the destination is unspecified but
+	positive. If the receiver has terminated, the signal does not arrive,
+	but it can trigger another signal. For example, a <c>link</c> signal
+	sent to a non-existing process triggers an <c>exit</c> signal, which
+	is sent back to where the <c>link</c> signal originated from. When
+	communicating over the distribution, signals can be lost if the
+	distribution channel goes down.
+      </p>
+
+      <p>
+	The only signal ordering guarantee given is the following: if an
+	entity sends multiple signals to the same destination entity, the
+	order is preserved; that is, if <c>A</c> sends a signal <c>S1</c> to
+	<c>B</c>, and later sends signal <c>S2</c> to <c>B</c>, <c>S1</c> is
+	guaranteed not to arrive after <c>S2</c>.
+      </p>
+    </section>
+
+    <section>
+      <marker id="sync-comm"/>
+      <title>Synchronous Communication</title>
+      <p>
+	At least when observed by the user, some communication is synchronous.
+	Such a communication can be broken down into multiple asynchronous
+	signals. An example of such a synchronous communication is a call to
+	the
+	<seemfa marker="erts:erlang#process_info/2"><c>erlang:process_info/2</c></seemfa>
+	BIF when the first argument does not equal the process identifier of
+	the calling process. The caller sends an asynchronous signal requesting
+	information, and then blocks waiting for the reply signal containing
+	the requested information. When the request signal reaches its
+	destination, the destination process replies with the requested
+	information.
+      </p>
+    </section>
+
+    <section>
+      <title>Irregularities</title>
+      <section>
+	<title>Synchronous error checking</title>
+	<p>
+	  Some functionality that send signals unfortunately have synchronous
+	  error checking when sending locally on a node and fail if the
+	  receiver is not present at the time when the signal is sent:
+	</p>
+	<list>
+	  <item>
+	    The <seeguide marker="expressions#send">send operator
+	    <c>!</c></seeguide>,
+	    <seemfa marker="erts:erlang#send/2"><c>erlang:send/2,3</c></seemfa>,
+	    BIFs and
+	    <seemfa marker="erts:erlang#send_nosuspend/2"><c>erlang:send_nosuspend/2,3</c></seemfa>
+	    BIFs when the receiver is identified by a name that is
+	    expected to be registered locally.
+	  </item>
+	  <item>
+	    <seemfa marker="erts:erlang#link/1"><c>erlang:link/1</c></seemfa>
+	  </item>
+	  <item>
+	    <seemfa marker="erts:erlang#group_leader/2"><c>erlang:group_leader/2</c></seemfa>
+	  </item>
+	</list>
+	<p>
+	  This has not been changed since it has been present so long.
```

Comment:
```suggestion
	  The irregularities mentioned above cannot be fixed as they have been part of Erlang too long and it would break a lot of existing code.
```